addons/account/migrations/12.0.1.1/pre-migration.py

```diff
@@ -132,6 +132,35 @@ def drop_obsolete_constraint_wizard_multi_charts_accounts(env):
     openupgrade.remove_tables_fks(env.cr, ['wizard_multi_charts_accounts'])
 
 
+def fix_double_membership(cr):
+    # avoid error raised by new function '_check_one_user_type'
+
+    # we arbitrarily keep the tax included
```

Comment:
Please keep tax excluded as B2B is the most usual one in my experience.

Reply:
My point was : B2B is the default, so if the user has b2c selected, it's intented. Dont't you think ?

Reply:
Is that really that way? Shouldn't all the users be homogeneous with one or the other? Applying res.config.settings apply to all users.